app/i18n/en/conf.php

```diff
@@ -162,6 +162,7 @@
 		'mark_read' => 'Mark as read',
 		'navigation' => 'Navigation',
 		'navigation_help' => 'With the "Shift" modifier, navigation shortcuts apply on feeds.<br/>With the "Alt" modifier, navigation shortcuts apply on categories.',
+		'navigation_no_mod_help' => 'The following navigation shortcuts do not support modifiers.',
```

Comment:
Please add the TODO's in the other languages as well

Reply:
Is there an automated way to do that?

Reply:
`sed` ? :-)

Reply:
lol, fair enough.

Reply:
Yeah, that worked well enough. Good thing you can reset quickly with Git to get the escaping just right though. ;-)

```
sed -i "/navigation_help/a\\\t\t'navigation_no_mod_help' => 'The following navigation shortcuts do not support modifiers.',\t//TODO - Translation" */conf.php
```

Reply:
Perfect :-)